cargo/Cargo.toml

```diff
@@ -8,17 +8,19 @@ license.workspace = true
 
 [dependencies]
 rustsec = { workspace = true }
-clap = { workspace = true, features = ["derive"] }
+clap = { workspace = true, features = ["derive", "unstable-doc", "wrap_help"] }
```

Comment:
Why does clap have the unstable-doc flag? I think it only adds extra documentation which should not be needed as an actual dependency.

Reply:
It's for `unwrap-help`. It's enabled with the `unstable-doc` 😅

Reply:
Wait i might be wrong 👀. i will try to build locally

Reply:
Woops I was wrong xD